src/memento_test.jl

```diff
@@ -3,6 +3,7 @@ module TestUtils
 using ..Memento
 using Test
 
+# Needed to extend these macros
```

Comment:
Is it needed?i thought you could  extend  macro without import

Reply:
If so I'm not sure what the syntax is cause I tried `Test.test_throws` and that errored.

Reply:
Huh. Tmyk